app/src/main/java/com/github/neighbortrader/foodboardapp/handler/clientmodelHandler/UserHandler.java

```diff
@@ -0,0 +1,114 @@
+package com.github.neighbortrader.foodboardapp.handler.clientmodelHandler;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.github.neighbortrader.foodboardapp.clientmodel.Address;
+import com.github.neighbortrader.foodboardapp.clientmodel.User;
+import com.github.neighbortrader.foodboardapp.handler.contextHandler.ContextHandler;
+import com.github.neighbortrader.foodboardapp.handler.gsonHandler.GsonHandler;
+import com.github.neighbortrader.foodboardapp.handler.requestsHandler.OnRequestEventListener;
+import com.github.neighbortrader.foodboardapp.handler.requestsHandler.RequestTyps;
+import com.github.neighbortrader.foodboardapp.handler.requestsHandler.UserRequestHandler;
+import com.google.gson.Gson;
+
+import java.util.UUID;
+
+import static android.content.Context.MODE_PRIVATE;
+
+public class UserHandler {
+
+    public static final String TAG = UserHandler.class.getSimpleName();
+
+    private static User userInstance;
+
+    private static final String SHARED_PREFERENCES_FILE_USER_INFO = "userInfo";
+
+    public static void setCurrentUserInstance(User userInstance) {
+        UserHandler.userInstance = userInstance;
+    }
+
+    public static User getCurrentUserInstance() {
+        return UserHandler.userInstance;
+    }
+
+    public static User generateRandomUser() {
+        String username = UUID.randomUUID().toString();
+        String password = "adm!n9Asswor!d";
+        String userId = null;
+        String email = "test@testemail.com";
+        Address address = new Address("Teststareße", "12a", "10315", "Berlin");
+
+        return User.userBuilder(username, password, userId, email, address, null);
+    }
+
+    public static void saveToSharedPreferences(User user) {
+        Log.d(TAG, "saveToSharedPreferences()");
+
+        if (user != null) {
+            Gson gson = GsonHandler.getGsonInstance();
+            // FIXME Caused by: java.lang.IllegalArgumentException: class java.text.DecimalFormat declares multiple JSON fields named maximumFractionDigits #23
+            String userToSaveAsJsonString = null; //gson.toJson(user);
+
+            SharedPreferences sharedPreferences = ContextHandler.getAppContext().getSharedPreferences(SHARED_PREFERENCES_FILE_USER_INFO, MODE_PRIVATE);
+            SharedPreferences.Editor editor = sharedPreferences.edit();
+
+            editor.putString(SHARED_PREFERENCES_FILE_USER_INFO, userToSaveAsJsonString);
+            editor.commit();
+        }
+    }
+
+    public static User loadFromSharedPreferences() {
```

Comment:
was wird hier geladen ? 
Das sollte noch mit in den Namen rein

Reply:
Ergibt sich ja eigentlich daraus das es im UserHandler ist ..!!!1!!

Warum hast du eigentlich immer eine Leerzeichen nach dem Satzende? Halt warte ?... besser !

Reply:
Dss erg8bt sich nicht automatisch.

Reply:
Auch bei dem Aufruf? `User newUser = UserHandler.loadFromSharedPrefrences();`

Aber es ist schon geändert